_posts/2024-01-26-two-months-in-servo.md

```diff
@@ -0,0 +1,146 @@
+
+## CI, code health, and dev changes
+
+The intermittent macOS build failures on CI should now be fixed ([@mrobinson](https://github.com/mrobinson), [#30975](https://github.com/servo/servo/pull/30975)).
+
+Servo now has some preliminary Android build support ([@mukilan](https://github.com/mukilan), [#31086](https://github.com/servo/servo/pull/31086)), though more work needs to be done before Servo will run on Android.
+
+The long-term effort to simplify how Servo is built continues ([@mrobinson](https://github.com/mrobinson), [#31075](https://github.com/servo/servo/pull/31075)), and we’ve replaced the `time` crate with `chrono` and `std::time` in much of Servo ([@Taym95](https://github.com/Taym95), [@augustebaum](https://github.com/augustebaum), [#30927](https://github.com/servo/servo/pull/30927), [#31020](https://github.com/servo/servo/pull/31020), [#30639](https://github.com/servo/servo/pull/30639), [#31079](https://github.com/servo/servo/pull/31079)).
+We have also started migrating our DOM bindings to use typed arrays where possible ([@gterzian](https://github.com/gterzian), [#30990](https://github.com/servo/servo/pull/30990), [#31077](https://github.com/servo/servo/pull/31077), [#31087](https://github.com/servo/servo/pull/31087), [#31076](https://github.com/servo/servo/pull/31076), [#31106](https://github.com/servo/servo/pull/31106)), as part of an effort to **reduce our unsafe code surface** ([#30889](https://github.com/servo/servo/issues/30889), [#30862](https://github.com/servo/servo/issues/30862)).
+
+Several crates have been moved into [our main repo](https://github.com/servo/servo):
+
+* `hyper_serde` ([servo/hyper_serde](https://github.com/servo/hyper_serde)) was moved to `components/hyper_serde` ([@mrobinson](https://github.com/mrobinson), [#30803](https://github.com/servo/servo/pull/30803))
+* `device` ([servo/devices](https://github.com/servo/devices)) was merged into `components/bluetooth` ([@mrobinson](https://github.com/mrobinson), [#30974](https://github.com/servo/servo/pull/30974))
+* `blurmac` ([servo/devices](https://github.com/servo/devices)) was moved to `third_party/blurmac` ([@mrobinson](https://github.com/mrobinson), [#30974](https://github.com/servo/servo/pull/30974))
+
+We’ve also made some dev changes:
+
+- the default branch in [our main repo](https://github.com/servo/servo) is now `main` ([@mrobinson](https://github.com/mrobinson), [@atouchet](https://github.com/atouchet), 23add0c1e5c9c, [#30877](https://github.com/servo/servo/pull/30877))
```

Comment:
I am not sure if just adding commit id `23add0c1e5c9c` is helpful. maybe we can add the link?

Reply:
oops, done in 0bbd3df3cc9feee69c5f5bf0e08dee219f4e840b